server/Cargo.toml

```diff
@@ -24,7 +24,7 @@ openssl                 = "0.10"
 ordered-float           = "3.2"
 prost                   = "0.11"
 prost-types             = "0.11"
-router                  = { git = "https://github.com/Arrow-air/lib-router.git", branch = "main" }
+router                  = { git = "https://github.com/Arrow-air/lib-router.git", branch = "roman/r2_time_ranges" }
```

Comment:
this can now be updated to develop branch

Reply:
I updated it